src/components/PeopleTable/UserInformation.jsx

```diff
@@ -0,0 +1,182 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useParams, useNavigate } from 'react-router-dom';
+import {
+  Container,
+  Text,
+  FormLabel,
+  Input,
+  FormControl,
+  GridItem,
+  SimpleGrid,
+  Image,
+  VStack,
+  Heading,
+  Grid,
+  Flex,
+  Box,
+  Select,
+} from '@chakra-ui/react';
+import { OCHBackend } from '../../common/utils';
+import AUTH_ROLES from '../../common/auth_config';
+
+const { ADMIN_ROLE, VOLUNTEER_ROLE } = AUTH_ROLES.AUTH_ROLES;
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'VOLUNTEER_ROLE' is assigned a value but never used.